system-variables.md

```diff
@@ -2666,6 +2666,18 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 默认值：`168h`，即 7 天
 - 这个变量用来控制历史统计信息在存储中的保留时间。
 
+### `tidb_idle_transaction_timeout` <span class="version-mark">从 v7.6.0 版本开始引入</span>
```

Comment:
To enhance user understanding and provide comprehensive guidance, could we consider adding the description of this system variable in the following documents?

https://github.com/pingcap/docs-cn/blob/7c86675626516d5bce7ddc210d183e9a0db6ce63/faq/manage-cluster-faq.md?plain=1#L64-L66

https://github.com/pingcap/docs-cn/blob/7c86675626516d5bce7ddc210d183e9a0db6ce63/troubleshoot-tidb-oom.md?plain=1#L123-L131

https://github.com/pingcap/docs-cn/blob/7c86675626516d5bce7ddc210d183e9a0db6ce63/develop/dev-guide-timeouts-in-tidb.md?plain=1#L7-L9

https://github.com/pingcap/docs-cn/blob/7c86675626516d5bce7ddc210d183e9a0db6ce63/develop/dev-guide-connection-parameters.md?plain=1#L238-L240

Reply:
@easonn7 PTAL

Reply:
It looks like these related documents need to be updated？

Reply:
Nice catch, done.